create-neon/src/template.ts

```diff
@@ -0,0 +1,12 @@
+import { readFile } from 'fs/promises';
+import handlebars from 'handlebars';
+import * as path from 'path';
+
+const TEMPLATES_DIR = path.join(__dirname, '..', 'data', 'templates');
+
+export default async function template(filename: string) {
+  let source = await readFile(path.join(TEMPLATES_DIR, filename), {
+    encoding: 'utf8'
+  });
+  return handlebars.compile(source, { noEscape: true });
```

Comment:
Not necessary, but it would be nice to precompile these and move handlebars to a `devDependency`.

Reply:
I don't think we can remove the dependency -- precompiling still results in a runtime dependency on handlebars. I'll still explore precompiling but I'm not sure there ends up being much payoff.

Reply:
It's been so long since I've used handlebars, I didn't recall there being a runtime. Yea, I guess it doesn't save much.

Reply:
Yeah I figured out I could do e.g.:

```sh
handlebars -c handlebars -n module.exports -f dist/src/templates/cargo-toml.js data/templates/Cargo.toml.hbs
```

for each template, but then we'd have to add that to the build command for every template, and still need the runtime dependency on `handlebars`. Nice idea but I think it doesn't end up working out.